### PR TITLE
Add stattrak extension

### PR DESCRIPTION
**Pull request by rafaelvicio**
## Description

Stattrak Esports
Everything you need about esports

- Esports tournaments: search for esports tournaments
- Esports matches: search for esports matches

## Screencast

Esports tournaments
Esports matches

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/stattrak/package.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "stattrak",
+  "title": "stattrak",
```
Comment (pernielsentikaer) [inline extensions/stattrak/package.json:4]:
```suggestion
  "title": "Stattrak",
```

File: extensions/stattrak/package.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    "esports",
+    "lol",
+    "valorant",
+    "cs"
+  ],
+  "commands": [
+    {
+      "name": "esports-tournaments",
+      "title": "Esports Tournaments",
+      "subtitle": "Current esports tournaments",
```
Comment (pernielsentikaer) [inline extensions/stattrak/package.json:25]:
```suggestion
      "title": "View Esports Tournaments",
      "subtitle": "Stattrak",
```

File: extensions/stattrak/package.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      "description": "search for esports tournaments",
+      "mode": "view"
+    },
+    {
+      "name": "esports-matches",
+      "title": "Esports Matches",
+      "subtitle": "Upcoming esports matches",
```
Comment (pernielsentikaer) [inline extensions/stattrak/package.json:32]:
```suggestion
      "title": "View Esports Matches",
      "subtitle": "Stattrak",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
- `useFetch` should cache the result but it looks empty while entering the extension, could you look into that? You can maybe do most of the sorting in `parseResponse` in `useFetch`

- Could you add at least one [screenshot](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) of the extension

**Comment by pernielsentikaer** [comment]
Is this ready for another review?

**Comment by rafaelvicio** [comment]
> r

Not yet, I'm working on some improvements.

**Comment by rafaelvicio** [comment]
> Is this ready for another review?

Not yet, I'm working on some improvements.

**Comment by pernielsentikaer** [comment]
Converting into a draft then 🙂

**Pull request closed without merging**